tests/python/test_scene.py

```diff
@@ -218,3 +220,37 @@ def test_transform_scenepart(box_f):
     bbox2 = get_bbox(box2)
 
     assert np.allclose(bbox1 + offset, bbox2)
+
+
+def test_ground_plane():
+    sp1 = ScenePart.from_obj("data/sceneparts/toyblocks/cube.obj")
+    sp1.rotate(axis=np.array([0, 0, 1.0]), angle=np.pi / 4)
+    sp1.scale(0.5)
+    sp1.translate(np.array([-45.0, 10.0, 10]))
+    sp1.force_on_ground = 1 #ForceOnGroundStrategy.LEAST_COMPLEX
```

Comment:
Just write `sp1.force_on_ground = ForceOnGroundStrategy.LEAST_COMPLEX` - that is the point of an enum after all.